<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-system"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="lwMenu">
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
       <item>
        <property name="text">
         <string>General</string>
        </property>
        <property name="toolTip">
         <string>General</string>
        </property>
        <property name="icon">
         <iconset theme="preferences-system">
          <normaloff>:/ide-icons/rc/Preferences-system-64.png</normaloff>:/ide-icons/rc/Preferences-system-64.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Style</string>
        </property>
        <property name="toolTip">
         <string> Style</string>
        </property>
        <property name="icon">
         <iconset theme="applications-graphics" resource="ide.qrc">
          <normaloff>:/ide-icons/rc/Mypaint-icon.png</normaloff>:/ide-icons/rc/Mypaint-icon.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_3">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tabEditorsettings">
            <attribute name="title">
             <string>Editor settings</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QPropertyGrid" name="propGridSettings"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTabWidget" name="tabWidget_2">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Editor</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPropertyGrid" name="propGridStyle"/>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCobolCodeEdit" name="codeEdit">
               <property name="plainText">
                <string>      *******************************************************************
      ** Example taken from http://progopedia.com/version/opencobol-1.0/*
      *******************************************************************
       IDENTIFICATION DIVISION.
      **************************************
       PROGRAM-ID. SAMPLE.
      **
       DATA DIVISION.
      **************************************
       WORKING-STORAGE SECTION.
      *-*-*-*-*-*-*-*-*-*-*-*-*-*
       77 FACT      PIC 9(15) comp  .
       77 N         PIC 99          .
       77 I         PIC 99          .
       77 IST       PIC XX          .
       77 FACTTST   PIC X(18)       .
      **
       PROCEDURE DIVISION.
      **************************************
       MAIN-PROCECURE.
           MOVE 16 to N
           MOVE 0 to I
           MOVE 1 to FACT
           PERFORM UNTIL I GREATER THAN N
               MOVE I to IST
               MOVE FACT to FACTTST
               DISPLAY IST &quot;! = &quot; FACTTST
               ADD 1 to I
               MULTIPLY I BY FACT
                    ON SIZE ERROR DISPLAY &quot;value too big&quot;
               END-MULTIPLY
           END-PERFORM
           EXIT PROGRAM.
       END PROGRAM SAMPLE.
      **
</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Console</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Background</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QColorButton" name="colorButtonConsoleBck">
                 <property name="color" stdset="0">
                  <color>
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Foreground</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QColorButton" name="colorButtonConsoleFore">
                 <property name="color" stdset="0">
                  <color>
                   <red>64</red>
                   <green>64</green>
                   <blue>64</blue>
                  </color>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>User input</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Application output</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QColorButton" name="colorButtonConsoleApp">
                 <property name="color" stdset="0">
                  <color>
                   <red>64</red>
                   <green>64</green>
                   <blue>255</blue>
                  </color>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QColorButton" name="colorButtonConsoleUsr">
                 <property name="color" stdset="0">
                  <color>
                   <red>34</red>
                   <green>170</green>
                   <blue>34</blue>
                  </color>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QInteractiveConsole" name="console"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Home page</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButtonWhite">
                 <property name="text">
                  <string>White</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButtonDark">
                 <property name="text">
                  <string>Dark</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QHomeWidget" name="homeWidget">
               <property name="title" stdset="0">
                <string>Welcome to OpenCoblIDE</string>
               </property>
               <property name="icon" stdset="0">
                <iconset resource="ide.qrc">
                 <normaloff>:/ide-icons/rc/silex-64x64.png</normaloff>:/ide-icons/rc/silex-64x64.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCodeEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>pyqode.core</header>
  </customwidget>
  <customwidget>
   <class>QPropertyGrid</class>
   <extends>QWidget</extends>
   <header>pyqode.widgets</header>
  </customwidget>
  <customwidget>
   <class>QHomeWidget</class>
   <extends>QWidget</extends>
   <header>pyqode.widgets</header>
  </customwidget>
  <customwidget>
   <class>QInteractiveConsole</class>
   <extends>QTextEdit</extends>
   <header>pyqode.widgets</header>
  </customwidget>
  <customwidget>
   <class>QColorButton</class>
   <extends>QWidget</extends>
   <header>pyqode.widgets</header>
  </customwidget>
  <customwidget>
   <class>QCobolCodeEdit</class>
   <extends>QCodeEdit</extends>
   <header>oci.editor</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ide.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
